src/bp/core/routers/bots/index.ts

```diff
@@ -164,6 +171,11 @@ export class BotsRouter extends CustomRouter {
     }
   }
 
+  private notifyChanges = (modification: FlowModification) => {
+    const payload = RealTimePayload.forAdmins('flow.changes', modification)
+    this.realtime.sendToSocket(payload)
+  }
```

Comment:
The router should only do its main job, which is to answer queries to specific endpoints and call services. It is also used to do some validation before calling the service.

Your calls to RealTimePayload should be done in the flow service, once the update is completed